src/engraving/libmscore/chordline.cpp

```diff
@@ -230,7 +230,7 @@ void ChordLine::write(XmlWriter& xml) const
     if (modified) {
         size_t n = path.elementCount();
         xml.stag("Path");
-        for (size_t i = 0; i < n; ++i) {
+        for (int i = 0; i < n; ++i) {
```

Comment:
For me, this causes a compiler warning about comparing a signed number with an unsigned number (int vs size_t). I think we should rather change PainterPath::elementAt() to use size_t instead of int.

Reply:
Possible, it fixed the warning with MSVC, I didn't cross-check this with MinGW (gcc).

Reply:
Better now?

Reply:
Yes, this one seems perfect now!